package.json

```diff
@@ -56,7 +56,7 @@
     "mapbox-gl": "^1.7.0",
     "module-alias": "^2.2.2",
     "mustache": "^4.0.0",
-    "openid-client": "^3.13.0",
+    "openid-client": "^3.11.0",
```

Comment:
**sigh** please add the `|| true` guard back to `ci:npm-outdated`

(we had to roll back this package due to errors when using the latest version alongside `esm`)